src/main/java/mate/academy/spring/service/mapper/MovieSessionMapper.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.model.dto.MovieSessionRequestDto;
+import mate.academy.spring.model.dto.MovieSessionResponseDto;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.MovieService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieSessionMapper {
+    private final MovieService movieService;
+    private final CinemaHallService cinemaHallService;
+
+    public MovieSessionMapper(MovieService movieService, CinemaHallService cinemaHallService) {
+        this.movieService = movieService;
+        this.cinemaHallService = cinemaHallService;
+    }
+
+    public MovieSessionResponseDto parse(MovieSession movieSession) {
```

Comment:
```suggestion
    public MovieSessionResponseDto toDto(MovieSession movieSession) {
```
same in other 2 mappers